build/fips/Dockerfile

```diff
@@ -0,0 +1,20 @@
+FROM us-docker.pkg.dev/google.com/api-project-999119582588/go-boringcrypto/golang:1.18.5b7 as build-fips
```

Comment:
Please add a short comment to this dockerfile - some people may get confused that this is used to build immudb docker image.

Reply:
ack, I will do that